### DISCUSSION TITLE
How to disable temporary SSR globally?

### DISCUSSION THREAD

**Discussion by danicruz0415** [solidjs/solid-start#1274] [Q&A]
Hi, 

I am experiencing some issues with SSR described here #1174.
While this is sorted out, I was wandering if I can disable SSR and run everything on the client

I found this part in [the documentation](https://start.solidjs.com/api/vite#reference) where it says that 

> ssr (boolean, default true): toggles between client rendering and server rendering (ssr) mode.

But when I set it as false, it errors saying that this property is not boolean but of type `SSROptions`
![image](https://github.com/solidjs/solid-start/assets/107332190/c8ceb469-9c58-4e0c-a2b3-4ebe6e2a9401)

I browse the interface, and it has the following properties: 

```
    noExternal?: string | RegExp | (string | RegExp)[] | true;
    external?: string[];
    target?: SSRTarget;
    optimizeDeps?: SsrDepOptimizationOptions;
    resolve?: {Object}
```

But none seem to indicate disabling the feature globally, as false did before. Is there a way to disable SSR globally and render everything client side only?
Thank you so much in advance.

PS: Using Solid start 0.4.9

**Comment by lxsmnsyc** [comment]
just do `start: { ssr: false }`

The docs mention "the exception of `start` property"

**Reply by trusktr** [reply]
I don't see a `start` property, but there's top-level `ssr` boolean these days.

```js
defineConfig({ssr: false, ...})
```

**Reply by lxsmnsyc** [reply]
@trusktr the answer was only valid pre-1.0
